src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder.cs

```diff
@@ -160,6 +160,9 @@ protected static bool IdentifiersMatch(ISyntaxFactsService syntaxFacts, string n
             FindReferencesDocumentState state,
             CancellationToken cancellationToken)
         {
+            if (!symbol.CanBeReferencedByName())
+                return ImmutableArray<FinderLocation>.Empty;
```

Comment:
this may break property get/set. n ot sure.  tests will let you know.  if tests pass, i'm good with this.

Reply:
I think this will also break indexers.

Reply:
➡️ Changed the condition to only look at anonymous types